reputation.php

```diff
@@ -546,9 +548,11 @@
 	{
 		error_no_permission();
 	}
+	
+	$plugins->run_hooks("reputation_delete");
```

Comment:
Hi, what about changing its name to `reputation_delete_end` ->

Reply:
Done!